pinot-query-planner/src/main/java/org/apache/pinot/query/QueryEnvironment.java

```diff
@@ -238,6 +240,11 @@ public List<String> getTableNamesForQuery(String sqlQuery) {
       Set<String> tableNames = RelToPlanNodeConverter.getTableNamesFromRelRoot(relRoot.rel);
       return new ArrayList<>(tableNames);
     } catch (Throwable t) {
```

Comment:
Not introduced in this PR, but we should not catch all `Throwable` here

Reply:
Maybe we should deprecate this `getTableNamesForQuery` method?